### PR TITLE
E-884 Feature: add staging env

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: .github/workflows/push-master.yml [hunk 1]
```diff
@@ -39,10 +39,10 @@ jobs:
       run: rm -rf ./dist
 
     - name: Build for PROD environment
-      run: yarn build
+      run: yarn build:staging
```
Comment (kazurus) [inline .github/workflows/push-master.yml:42]: Where do we build prod dist?

Comment (dudiq) [reply .github/workflows/push-master.yml:42]: did not read correctly this conf =/
add commit with fix

**Comment by kazurus** [comment]
approved

**Pull request merged**